### PR TITLE
add incorrect link string

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: src/ko.json [hunk 1]
```diff
@@ -1532,7 +1532,8 @@
             "insufficientBalanceForFee": "네트워크 수수료를 지불할 금액이 부족합니다",
             "insufficientFundsToActivate": "프로필 활성화할 자금이 부족합니다",
             "limitReached": "금액이 설정된 한도 중 하나를 초과합니다.",
-            "notYetSupported": "아직 지원되지 않음"
+            "notYetSupported": "아직 지원되지 않음",
+            "incorrectLink": "Unfortunately, the everlink is not correct; please get in touch with the third-party service developers."
```
Comment (sertony) [inline src/ko.json]: Let's ask @kkkhesha to translate

File: src/ru.json [hunk 2]
```diff
@@ -1530,7 +1530,8 @@
             "insufficientBalanceForFee": "Недостаточно средств для оплаты комиссии сети",
             "insufficientFundsToActivate": "Недостаточно средств для активации профиля",
             "limitReached": "Сумма превышает один из установленных лимитов",
-            "notYetSupported": "Пока не поддерживается"
+            "notYetSupported": "Пока не поддерживается",
+            "incorrectLink": "Unfortunately, the everlink is not correct; please get in touch with the third-party service developers."
```
Comment (sertony) [inline src/ru.json]: Let's ask @kkkhesha to translate

File: src/ko.json (hunk 1 above)
Comment (kkkhesha) [inline src/ko.json]:
```suggestion
            "incorrectLink": "불행하게도 에버링크가 잘못되었습니다. 타사 서비스 개발자에게 문의하십시오."
```

File: src/ru.json (hunk 2 above)
Comment (kkkhesha) [inline src/ru.json]:
```suggestion
            "incorrectLink": "К сожалению, возникла ошибка с вашей эверссылкой. Пожалуйста, обратитесь к разработчику данного сервиса."
```

**Pull request merged**